### PR TITLE
ACMS-644: Enable Drupal 9 & 10 compatible release of GDPR with securi…

### PR DESCRIPTION
**Pull request by chandan-singh7929**
…ty coverage.

**Motivation**

Fixes #[ACMS-644](https://backlog.acquia.com/browse/ACMS-644)

**Proposed changes**
Provides GDPR implementation using acquia_cms_common module

**Alternatives considered**
Use acquia_cms_privacy module

**Testing steps**

- Checkout to branch ACMS-644
- Run `composer require drupal/gdpr drupal/eu_cookie_compliance drupal/gdpr_fields`
- Install site by running `./vendor/bin/acms si --account-pass admin`
- Enable GDRP modules by running command `./vendor/bin/drush en gdpr eu_cookie_compliance gdpr_fields`
**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_common/config/optional/gdpr_fields.gdpr_fields_config.node.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+langcode: en
```
Comment (rajeshreeputra) [inline modules/acquia_cms_common/config/optional/gdpr_fields.gdpr_fields_config.node.yml:1]: Please move this file to ACMS Person module

File: modules/acquia_cms_common/composer.json [hunk 2]
```diff
@@ -100,6 +100,11 @@
             },
             "drupal/focal_point": {
                 "3162210 - Preview link accidentally closes the media library": "https://www.drupal.org/files/issues/2020-10-06/3162210-17.patch"
+            },
+            "drupal/checklistapi": {
+                "Alter the prgress bar based upon the default value passed": "https://www.drupal.org/files/issues/2021-10-26/checklistapi-count-default-values-forprogress-2311855-2.patch",
```
[the remark this answers is not in the document]
Comment (chandan-singh7929) [reply modules/acquia_cms_common/composer.json]: Done

File: modules/acquia_cms_common/config/optional/gdpr_fields.gdpr_fields_config.node.yml (hunk 1 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_common/config/optional/gdpr_fields.gdpr_fields_config.node.yml]: Done

**Review by rajeshreeputra — APPROVED** [review]
Changes Looks Good

**Review by apathak18 — APPROVED** [review]

**Review by deepakmishra2 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**